CMakeLists.txt

```diff
@@ -354,7 +354,7 @@ set(_versionInfoHeader
 #define HIP_VERSION       (HIP_VERSION_MAJOR * 100 + HIP_VERSION_MINOR)\n
 #endif\n
 ")
-file(WRITE "${CMAKE_CURRENT_SOURCE_DIR}/include/hip/hipVersion.h" ${_versionInfoHeader})
+file(WRITE "${CMAKE_CURRENT_SOURCE_DIR}/include/hip/hip_version.h" ${_versionInfoHeader})
```

Comment:
There is a suggestion of not generating the file under the source folder but to generate it under the build folder so that git will not track it as new addition to the repro.

Reply:
The way to handle this is to add it to .gitignore similar to what we have done for include/hip/hcc_detail/hip_prof_str.h (which is also a generated file)